coreos-postinst

```diff
@@ -226,9 +226,15 @@ good_sysctl() {
     [[ -n "${GOOD_SYSCTL_SUMS[${sum}]}" ]] && return 0
 }
 
+# Gather metadata about upcoming OS image.
+# shellcheck source=/dev/null
+source "${INSTALL_MNT}/lib/os-release"
+NEXT_VERSION_ID=${VERSION_ID}
+
 # Check to make sure that the current version is less than 1576 (this is the
 # first release to include this Tectonic work-around). This ensures that we
 # only apply this work-around one time.
+# shellcheck source=/dev/null
```

Comment:
Was this line (and the one above) added intentionally?  I don't see this used in any of our other scripts.

Reply:
This is to prevent shellcheck from trying to follow it: https://github.com/koalaman/shellcheck/wiki/SC1090